UOP1_Project/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -5,7 +5,7 @@ EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Scenes:
-  - enabled: 1
-    path: Assets/Scenes/SampleScene.unity
-    guid: d1c3109bdb54ad54c8a2b2838528e640
+  - enabled: 0
```

Comment:
should probably not touch this file

Reply:
Ah thought I reverted that one sec. Surprised it's not in the git ignore good catch!